docs/getting-started-guides/locally.md

```diff
@@ -129,6 +132,30 @@ cluster/kubectl.sh create -f docs/user-guide/pod.yaml
 
 Congratulations!
 
+### Hacking hack-local-up cluster to test alternative deployments and options.
```

Comment:
Sorry to be debby downer, but I recommend removing this whole section. There's probably something useful to be said about certs and authentication, but I think it should go in a more general place, since it's not specific to the local up script, and it should probably have some instructions about how to generate a self signed cert for testing. It may already exist, too.